.github/workflows/sync-labels.yml

```diff
@@ -0,0 +1,21 @@
+name: Tools
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - .github/labels.yml
+      - .github/workflows/sync-labels.yml
+  schedule:
+    - cron: 0 0 * * 1
```

Comment:
This runs more often than before with `6 6 6 * *`. I don't think that's really needed?

Reply:
It's just once a week, which I felt was a nice compromise between your frequence and Elixir's frequency: https://github.com/exercism/elixir/blob/main/.github/workflows/sync-labels.yml 🤷

Reply:
A label sync build takes ~45s each. Across 284 repos, those extra 3 builds a month add up to ~10 CI hours extra a month. That seems excessive for something that is only meant to catch freak accidents in case a `push`  somehow doesn't trigger a rebuild.